src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private int[] NUMBERS = new int[12];
```

Comment:
This is an array, not a constant (should be written lowercase)
Constants not extracted. 
Maybe you have forgotten to merge PR from Misztal?